README.md

```diff
@@ -31,217 +49,213 @@ wiki
 
-1. Navigate to your system's root folder and install command-line-tools 
-    > xcode-select --install
+## Uploading Images
 
-    > export SDKROOT=$(xcrun --show-sdk-path)
+To add an image to any wiki page:
+1. Place the image file within the most appropriate subfolder under `docs/_assets/`.
+2. Name the image file with an intuitive name. Many existing wiki images are named `[topic]_[detail].png`, such as: `readme_nav-example.png`.
+3. Insert the image into your webpage with the appropriate markdown:
 
-2. Install Homebrew by following the instructions on the [Homebrew homepage](https://brew.sh/) to paste a specific command into your Terminal.
-
-3. Install Ruby using homebrew
-    > brew install ruby
-
-    > echo 'export PATH="/usr/local/opt/ruby/bin:/usr/local/lib/ruby/gems/3.0.0/bin:$PATH"' >> ~/.bash_profile
-
-5. Install bundler and jekyll by running:
-    > gem install --user-install bundler jekyll
+```yml
+![image-file-name](https://raw.githubusercontent.com/NDCLab/wiki/main/image-file-path)
+```
 
-6. Navigate to your local `wiki` repository and initialize the repo.
-    > cd wiki
-    > bundle init 
+For example, the image that appears above (showing the wiki homepage) is called in this markdown file as:
 
-7. This will create a gemfile called `Gemfile`. Add the following to this newly created file: 
-    ```ruby
-    gem "just-the-docs"
-    gem "jekyll-remote-theme"
-    ```
-
-8. Install the newly listed gems using: 
-    > bundle install
+```yml
+![readme_homepage](https://raw.githubusercontent.com/NDCLab/wiki/main/docs/_assets/technical/readme_homepage.png)
+```
 
-9. Add the following block anywhere in the `_config.yml` file
-    ```yml
-    plugins:
-        - jekyll-remote-theme
-    ```
+This method for linking images has two key benefits:
+1. The image file is easily accessible within the wiki repo.
+2. Given that the `main` branch is the published branch, the image displays correctly upon publication.
 
-10. Run the following command to build the website on your local machine. 
-    > bundle exec jekyll serve
+However, this method does have one major disadvantage of which users should be aware: the image will **not** render correctly on their working branch until it is merged with `main`. As a result, users are encouraged to double-check all images after the merge with `main` to ensure they are displaying as intended.
 
-### Page Creation 
+## Adding Pages
 
-To create a page on the wiki and link it to the left-hand navigation bar, create the markdown file inside of the [docs/](https://github.com/NDCLab/wiki/tree/gh-pages/docs) directory and specify its' position on the navbar in the header.  
+To add a brand new page on the wiki, create a new markdown file inside of the appropriate directory. This markdown file should begin with the following content, which specifies its name and nav order in the wiki structure:
```

Comment:
i dont think markdown was defined (and/or linked to a definition and cheat sheet) on this page.

Reply:
@F-said @jessb0t

Reply:
@georgebuzzell Do you feel that this is needed, given that **all** readmes on GitHub use markdown by necessity?  Note, also, that Get with Git explicitly contains a whole module on Markdown, offered directly from GitHub.

Happy to add, I just want to make sure we don't clutter with too many basics and I think knowledge of Markdown will be something that people already have before they poke around this readme.  But I could be wrong!

Reply:
I think I am just looking to a link to the page where it is defined is all

@jessb0t

Reply:
It is here!
https://ndclab.github.io/wiki/docs/technical-docs/git_and_github.html#markdown